types/@gavant/ember-floating-labels/fl-input.d.ts

```diff
@@ -0,0 +1,14 @@
+import Component from '@glint/environment-ember-loose/glimmer-component';
+
+import { BaseGlimmerSignature, ModifyYields } from '@gavant/glint-template-types/utils/types';
+
+interface FlInputYields {
+    Yields: {
+        default: any;
+    };
+}
+
+import { FlInputArgs } from '@gavant/ember-floating-labels/addon/components/fl-input';
+export default class FlInputComponent extends Component<
```

Comment:
It looks like in most other type files, we don't append Component on. Can you remove that for this file and anything under the @gavant directory

Reply:
Im not concerned about this being a breaking change because for the most part, its just us who use it.